packages/widgets/src/bookmarks/bookmark.module.css

```
@@ -23,3 +23,15 @@
 .bookmarkButton:focus-visible .bookmarkIcon {
   background-color: var(--mantine-color-iconColor-filled-hover);
 }
+
+.compactScrollArea :global(.mantine-ScrollArea-viewport) > div {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)